integration/VODE/vode_type_simplified_sdc.H

```diff
@@ -0,0 +1,668 @@
+  //  SFS-1+nspec |   ...                                                                         |
+  //  SENTH       | d(rho h)/drho      d(rho h)/dX1       d(rho h)/dX2       ...  d(rho h)/dT     |
+  //  SRHO_EXTRA  \ 0                  0                  0                       0               /
+
+
+  // fill the column of dRdw corresponding to the derivative
+  // with respect to rho
+  for (int m = 1; m <= NumSpec; m++) {
+    // d( d(rho X_m)/dt)/drho
+    dRdw(SFS+m, iwrho) = ydot(m) + state.rho * (ydot_pert(m) - ydot(m))/(eps * state.rho);
+  }
+
+  // d( d(rho h)/dt)/drho
+  dRdw(SENTH+1, iwrho) = ydot(net_ienuc) +
+    state.rho * (ydot_pert(net_ienuc) - ydot(net_ienuc))/(eps * state.rho);
+
+  // d( d(rho)/dt)/drho
+  dRdw(SRHO_EXTRA+1, iwrho) = 0.0_rt;
+
+  // fill the columns of dRdw corresponding to each derivative
+  // with respect to species mass fraction
+  for (int n = 1; n <= NumSpec; n++) {
+    for (int m = 1; m <= NumSpec; m++) {
+      // d( d(rho X_m)/dt)/dX_n
+      dRdw(SFS+m, iwfs-1+n) = state.rho * jac_react(m, n);
+    }
+
+    // d( d(rho h)/dt)/dX_n
+    dRdw(SENTH+1, iwfs-1+n) = state.rho * jac_react(net_ienuc, n);
+
+    // d( d(rho)/dt)/dX_n
+    dRdw(SRHO_EXTRA+1, iwfs-1+n) = 0.0_rt;
+  }
+
+  // now fill the column corresponding to derivatives with respect to
+  // temperature -- this column is iwT
+
+  // d( d(rho X_m)/dt)/dT
+  for (int m = 1; m <= NumSpec; m++) {
+    dRdw(SFS+m, iwT) = state.rho * jac_react(m, net_itemp);
+  }
+
+  // d( d(rho h)/dt)/dT
+  dRdw(SENTH+1, iwT) = state.rho * jac_react(net_ienuc, net_itemp);
+
+  // d( d(rho)/dt)/dT
+  dRdw(SRHO_EXTRA+1, iwT) = 0.0_rt;
+
+  // that completes dRdw
+
+  // construct dwdU.  Here we take U = (rho X, rho h, rho)^T
+
+  // density row (iwrho)
+  dwdU(iwrho, SRHO_EXTRA+1) = 1.0_rt;
+
+  // species rows
+  for (int m = 1; m <= NumSpec; m++) {
+    dwdU(iwfs-1+m, SFS+m) = 1.0_rt / state.rho;
+    dwdU(iwfs-1+m, SRHO_EXTRA+1) = -state.xn[m-1] / state.rho;
+  }
+
+  // T row
+  eos_t eos_state;
+  eos_state.rho = state.rho;
+  eos_state.T = 1.e4;   // initial guess
+  for (int n = 0; n < NumSpec; n++) {
+    eos_state.xn[n] = vode_state.y(SFS+n) / state.rho;
+  }
+#if NAUX_NET > 0
+  amrex::Error("error: aux support not implemented");
+#endif
+  eos_state.h = vode_state.y(SENTH+1) / state.rho;
+
+  eos(eos_input_rh, eos_state);
+
+  const eos_xderivs_t eos_xderivs = composition_derivatives(eos_state);
+
+  // temperature row
+  for (int n = 1; n <= NumSpec; n++) {  
+    dwdU(iwT, SFS+n) = -eos_xderivs.dhdX[n-1] / (state.rho * eos_state.dedT);
+  }
+  dwdU(iwT, SENTH+1) = 1.0_rt / (state.rho * eos_state.dhdT);
+  Real X_dedX_sum = 0.0;
+  for (int n = 0; n < NumSpec; n++) {
+    X_dedX_sum += eos_state.xn[n] * eos_xderivs.dedX[n];
+  }
+  dwdU(iwT, SRHO_EXTRA+1) =
+    (X_dedX_sum - state.rho * eos_state.dhdr - eos_state.h) /
+    (state.rho * eos_state.dhdT);
```

Comment:
```suggestion
  Real X_dhdX_sum = 0.0;
  for (int n = 0; n < NumSpec; n++) {
    X_dhdX_sum += eos_state.xn[n] * eos_xderivs.dhdX[n];
  }
  dwdU(iwT, SRHO_EXTRA+1) =
    (X_dhdX_sum - state.rho * eos_state.dhdr - eos_state.h) /
    (state.rho * eos_state.dhdT);
```

I think this should use `dhdX` here instead of `dedX`